src/permissions/rules/rules.service.spec.ts

```diff
@@ -44,33 +53,52 @@ describe('RulesService', () => {
       ],
     }).compile();
 
-    service = module.get<RulesService>(RulesService);
+    service = module.get(RulesService);
   });
 
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
 
-  it('should fetch the rules from the db on startup', async () => {
+  it('should fetch the rules from the db on startup', () => {
     expect(mockCouchDBService.get).toHaveBeenCalledWith(
       DATABASE_NAME,
-      Permission.DOC_ID,
+      '_changes',
+      expect.anything(),
     );
```

Comment:
is this a relevant test? I think the call to CouchDB is not triggering anything important, rather what we may want to check here is whether the ruls in our RulesService have be correctly initialized?